### DISCUSSION TITLE
Can RaspiBlitz be setup on a RPI 3 B+ in headless mode with no keyboard/screen (ssh + headless)

### DISCUSSION THREAD

**Discussion by LlamaPony** [raspiblitz/raspiblitz#4022] [Q&A]
Greetings.
I hope whoever is reading this is doing well and having an epic morning/day/eve.

Can RaspiBlitz be setup on a RPI 3 B+ in headless mode with no keyboard/screen (ssh + headless) ?

**Comment by rootzoll** [comment]
The RPi3 is quite limited in process power and the USB2 is quite slow to do blockchain operations with the ssd. You can try by downloading the "Minimal Image" which starts headless by default. I would then recommend just to run the basics (no additional apps, maybe even no lightning) and sync the blockchain from a laptop or other raspiblitz.

If you try this - le us know your results.
